.env.test

```diff
@@ -0,0 +1,14 @@
+# Database
+DATABASE_URL=postgresql://postgres@localhost:5432/payments_alpha_test
```

Comment:
Not sure how I feel about this because it ties you to this DB URL and you'd have to locally ignore your copy of `.env.test` if you want to change this. Let's sleep on it and think about it again on Monday.